src/components/itemhelper.js

```diff
@@ -273,8 +273,13 @@ define(['apphost', 'globalize'], function (appHost, globalize) {
         },
 
         canRate: function (item) {
+            var itemType = item.Type;
+
+            if (item.Type === 'Program' || item.Type === 'Timer' || item.Type === 'SeriesTimer' || item.Type === 'CollectionFolder' || item.Type === 'UserView' || item.Type === 'Channel' || itemType === 'Season' || itemType === 'Studio') {
+                return false;
+            }
```

Comment:
There's no need to create a new variable here, the other checks just use the item type directly.

Reply:
need by season and studio item.Type not define.  if i remove those to yay it not need.  otherwise we can add season on the favorite section